### ISSUE TITLE
unable to import backup file from Brave Payments

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#1325]
## Description 
When attempting to import the recovery file from Brave Payments (which contains 24 backup words), I get an error message that says 'Please re-enter keys or try different keys'. Copy/Paste of words works fine.

## Steps to Reproduce

Pre-req: if you don't have a recovery file for Brave Payments, open latest b-l version and enable Brave payments. Create a recovery file from your created wallet.
   1.  Launch 0.55.10 (this uses prod wallet).
   2. If not enabled, enable Brave Rewards.
   3. From chrome://rewards click on the gear icon in the wallet panel/container.
   4. Click Restore your Wallet.
   5. Click on import link.
   6. Navigate to your recovery file.

## Actual result:
Error message displayed
![screen shot 2018-09-27 at 3 50 07 pm](https://user-images.githubusercontent.com/28145373/46170879-07082c00-c26d-11e8-822b-16df954d0e94.png)

Copy/paste of same words works.

## Expected result:
No error. File should be imported and wallet restored.

## Reproduces how often: 
easily

## Brave version (chrome://version info)

Brave | 0.55.10 Chromium: 70.0.3538.22 (Official Build) beta(64-bit)
-- | --
Revision | ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364-refs/branch-heads/3538@{#468}
OS | Mac OS X

### Reproducible on current release:
no

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

### Additional Information
cc @brave/legacy_qa to check on other platforms

**Comment by NejcZdovc** [comment]
@LaurenWags did you tried this on beta?

**Comment by NejcZdovc** [comment]
so the problem is just with file import right? not with regular copy/paste of keys?

**Comment by LaurenWags** [comment]
@NejcZdovc yes, used Beta build version 0.55.10. And yes, issue is just with import of file. Copy/Paste keys works as expected.

**Comment by kjozwiak** [comment]
Basically seeing the same thing as @LaurenWags on all three platforms. When I attempt to import my wallet via a `brave_wallet_recovery.txt` file that I generated with b-l, I get the following error message every single time:

* `Please re-enter keys or try different keys.`

However, when I import via copying/pasting they keys themselves, it restores without any issues.

Used the following build using `macOS 10.13.x 64`, `Win 10 x64` & `Ubuntu 18.04 x64`:

Brave | 0.55.10 Chromium: 70.0.3538.22 (Official Build) beta(64-bit)
-- | --
Revision | ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364-refs/branch-heads/3538@{#468}
OS | Mac OS X

**Comment by srirambv** [comment]
Verification Passed on 

Brave | 0.55.12 Chromium: 70.0.3538.45 (Official Build) beta (64-bit)  
-- | -- 
Revision | cbdc32e4334458954e9def214d7e5fa1ca1960eb-refs/branch-heads/3538@{#830}
OS | Linux

![import wallet](https://user-images.githubusercontent.com/17010094/46756616-de812880-cc95-11e8-8237-b07493c0e619.gif)

**Comment by fscchan** [comment]
Adding to this. I tried it 3 times on 3 different computers and 3 different OS.

- [Windows 7](https://i.imgur.com/AvQAdVy.mp4)
- [Windows 10](https://i.imgur.com/mdOfOSA.mp4)
- [Manjaro](https://i.imgur.com/3E8LgPE.mp4)
